definitions/ext-host/definition.yml

```diff
@@ -10,9 +10,20 @@ synthesis:
         - attribute: metricName
           prefix: datadog.system.
 
+    # telemetry from generic flex agent
+    - identifier: displayName
+      name: displayName
+      encodeIdentifierInGUID: true
+      conditions:
+      - attribute: eventType
+        value: FlexSystemSample
+
 dashboardTemplates:
-  newRelic:
-    template: dashboard.json
+  flexagent:
+    template: flex-dashboard.json
+  datadog:
+    template: datadog-dashboard.json
+
 configuration:
-  entityExpirationTime: DAILY
+  entityExpirationTime: QUARTERLY
```

Comment:
Im leaning on keeping it as daily. that's also the default in INFRA-HOST: https://github.com/newrelic/entity-definitions/blob/main/definitions/infra-host/definition.yml#L46

so making every host the same expiration makes sense to me and is easier on future merges.

what do you think?

Reply:
Needs to be QUARTERLY for this scenario: User runs the Flex agent temporarily to harvest data for a period of time, it will not run indefinitely as during migrations they just need to capture for a short period to understand the environment and entities within it.
Migrations can take weeks to months, so the entity needs to live far longer then a day.

Reply:
hmm that makes sense. i was wondering if this should go under infra-host but that seems a rather specific case good enough to be on ext-host